diversity-lunch-be/src/test/java/de/brockhausag/diversitylunchspringboot/integrationstests/EMailControllerIT.java

```diff
@@ -131,14 +131,33 @@ public void testSendTestMail_expectedCorrectTestContentInMail() {
 
     @SneakyThrows
     @Test
-    public void testSendTestMailToUser_expectedToNotThrowException() {
+    public void testSendTestMailToUser_withValidId_expectedOkStatus() {
+        when(this.diversityLunchMailProperties.getSender()).thenReturn("diversitylunchtest@brockhaus-ag.de");
         ProfileEntity tmpProfileEntity = profileTestdataFactory.createEntity();
-        var accountEntity1 = accountService.getOrCreateAccount(tmpProfileEntity.getEmail());
-        long id = 42;
+        AccountEntity accountEntity1 = accountService.getOrCreateAccount(tmpProfileEntity.getEmail());
```

Comment:
You don't have to write "accountEnitity1" it should be "accountEnitity" same in the next section and tests